server/src/com/cloud/network/NetworkServiceImpl.java

```diff
@@ -3085,9 +3085,10 @@ public GuestVlan dedicateGuestVlanRange(DedicateGuestVlanRangeCmd cmd) {
         // Verify account is valid
         Account vlanOwner = null;
         if (projectId != null) {
-            if (accountName != null) {
-                throw new InvalidParameterValueException("accountName and projectId are mutually exclusive");
-            }
+            //accountName and projectId are mutually exclusive
```

Comment:
@koushik-das : As I mentioned in resolution part, Even if the account name is passed while adding from a particular project view, then it is ignored(Removal part) in API. If we don't remove that validation check, then the same issue will happen again.